python/fixrgraph/musedev/api.py

```diff
@@ -123,16 +134,54 @@ def finalize(cmd_input):
     residue = get_none(cmd_input.json_input, "residue")
 
     # Example: loop through the compilation info
+    javafiles = []
     for compilation_info in Residue.get_compilation_infos(residue):
         for filePath in Residue.get_files(compilation_info):
-            pass
+            if filePath.endswith(".java"):
+                javafiles.append(filePath)
+
+    # extract the graphs
+    extractor_jar = os.getenv("GRAPHEXTRACTOR")
+    try:
+        graphdir = tempfile.mkdtemp(".groum_test_extract_single")
+
+        # To call directly, uncomment the following lines
+        #TODO: get github org and repo name
+        extract_single.extract_single_class_dir(repo = ["unkown","unknown",cmd_input.commit],
```

Comment:
@smover, For now, I use the organization and app name as "unknown" and "unknown" which seems to work. I would like to get this from somewhere, perhaps the muse custom tool API.